internal/translation/dbuser/conversion.go

```diff
@@ -0,0 +1,221 @@
+package dbuser
```

Comment:
this package is missing unit tests.

Reply:
I can add conversion unit tests, but the contract tests are the ones that should cover the package fully.

Reply:
Done